### DISCUSSION TITLE
What's the difference between "App Left Half | App Right Half" and "Left Half | Right Half"?

### DISCUSSION THREAD

**Discussion by cori** [rxhanson/RectanglePro-Community#307] [Q&A] [answered]
As it says on the tin I'm not clear on what the difference between these two actions are.

Maybe a better question is whether this is the right venue for this question. Seems like I should be able to RTFM to understand this, but had no luck searching for one.

**Comment by rxhanson** [comment, marked as answer]
Thanks for asking.

App left/right half is "Move all windows of this app to the left half or right half".

Left/right half is "Move this window left half or right half"

There is no manual (this is it), but I'll add in an info button on those actions so it's more clear from within the app.

**Reply by cori** [reply]
That's sort of what i inferred, both about the command and the manual, but didn't want to assume. Thanks!
